openlr/openlr_match_quality/openlr_assessment_tool/main.cpp

```diff
@@ -29,8 +29,6 @@ int main(int argc, char * argv[])
 
   Q_INIT_RESOURCE(resources_common);
   QApplication app(argc, argv);
-  // Pretty icons on HDPI displays.
-  QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
```

Comment:
Well, I'd prefer to make it (and similar below) in a separate commit in case we will revert Qt5 back.

Reply:
Done. Don't squash commits when merge